judgments/tests/template_tags/test_query_filters.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+
+from judgments.templatetags.query_filters import replace_year_in_query
+
+
+class TestQueryFilters(unittest.TestCase):
+    def test_replace_year_in_query(self):
+        query_params = {
+            "from": "",
+            "from_day": "3",
+            "from_month": "9",
+            "from_year": "2009",
+            "to": "",
```

Comment:
we're modifying "to" and "from" so we should probably test they're removed/blanked as well.